Presentation.Web/Swagger/OnlyIncludeReadModelSchemasInSwaggerDocumentFilter.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Swashbuckle.Swagger;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http.Description;
+using Core.DomainModel;
+
+namespace Presentation.Web.Swagger
+{
+    public class OnlyIncludeReadModelSchemasInSwaggerDocumentFilter : IDocumentFilter
```

Comment:
Since we intend to nuke every other odata controller, and we must customize the output anyways, this filter ensures we only get output models for read models